_posts/2023-10-05-dask-expr-tpch-dask.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+date: 2023-10-05
+tags: [dask, query optimizer, performance, benchmarks]
+author: JPatrick Hoefler
+theme: twitter
+canonical_url: https://blog.coiled.io/blog/dask-expr-tpch-dask.html
+---
+
+# TPC-H Benchmarks for Query Optimization with Dask Expressions
+
+<a href="/images/dask_expr/tpch-comparison-dask-confidence.png">
+<img src="/images/dask_expr/tpch-comparison-dask-confidence.png"
```

Comment:
1. This figure is never referred to in the text, did I miss that?
    I think there should be a small paragraph or caption explaining what the figure shows. This figure was also the least intuitive for me to understand out of the three, so I think it's especially important to explain. (i.e. what is the x-axis, it's the confidence threshold of what? Of the wall clock time? That's the tab that is selected, but the graph itself doesn't have the axis labelled with what it actually measured)
2. This figure and text really belong in the results section. I think it's here because you want the readers to see something pretty when they click on the blog post. I think it's ok to have a copy of a pretty figure used as an illustration up the top (but you still need the figure and text to exist in the results section below).